client/src/test/java/com/sendwithus/SendWithUsTest.java

```diff
@@ -187,6 +187,24 @@ public void testSendWithVersionName() throws SendWithUsException
     }
 
     /**
+     * Test send with empty esp account
+     */
+    @Test
+    public void testSendWithLocale() throws SendWithUsException
+    {
+        String locale = "";
```

Comment:
Technically this will work because an empty local parameter will use the default locale, but you should put `en-US` in there.